webpack.config.renderer.js

```diff
@@ -42,6 +43,19 @@ module.exports = merge(base, {
       ignoreOrder: true,
       chunkFilename: '[id].[contenthash].css',
     }),
+    new CopyPlugin({
+      patterns: [{
+        from: 'assets/windows/*.ico',
+        context: 'src',
+      }, {
+        from: 'assets/linux/*/*.png',
```

Comment:
I think you can reduce this by doing:
```suggestion
        from: 'assets/**/*.png',
```